### PR TITLE
[CONFIG] dockerization for Dokku deployment

### PR DESCRIPTION
**Pull request by robturtle**
- Dockerfile
- pin the nodejs version to latest
- bump version to 0.2.0

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM node:10 As builder
```
Comment (wangtjwork) [inline Dockerfile]: why here it's v10?

Comment (robturtle) [reply Dockerfile]: good catch!

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+RUN yarn build
+
+FROM nginx:alpine As host
+
+# You can even configure more if you use a nginx.conf.sigil file
+# (http://dokku.viewdocs.io/dokku/configuration/nginx/#customizing-the-nginx-configuration)
+COPY nginx.default.conf /etc/nginx/conf.d/default.conf
+
+WORKDIR /usr/share/nginx/html
+COPY --from=builder /app/build .
```
Comment (yc-shawn) [inline Dockerfile:21]: Can you explain the this line?

The built files are stored in `./build`

image

Comment (wangtjwork) [reply Dockerfile:21]: There are two containers in this process. The first one is `FROM node:14 AS builder`, second one is `FROM nginx:alpine AS host`. Line 3 states that we use /app as the root directory for all commands in **builder** container, so all these `ADD` and `RUN` commands are executed inside /app folder.
Therefore the files produced by `RUN yarn build` exists under **builder** container's /app folder, which is `/app/build`.

Comment (robturtle) [reply Dockerfile:21]: 噗~ Docker 是跑在容器里的啊。其中 builder 的工作目录被设置成 /app 了啊，那在 /app 下面build出来的目录不就在 /app/build 下了嘛

**Pull request merged**